coordinape/coordinape
# Fixes #440 by implementing hasura action and FE changes for circle lo…

Fixes #440 by implementing hasura action and FE changes for circle logo upload. Currently targeting merge into 435-hasura-upload-avatar but will move to main after that is merged. 

Test plan after deploy to staging/local:

set 
`REACT_APP_HASURA_ENABLED=true`
in .env to enable this. It's disabled by default so it doesn't disrupt production. 

1. visit the admin page  for a circle
2. click Edit settings
3. click the pencil icon on the circle logo
4. choose an image
5. click save at the bottom of the modal dialog
6. make sure it finishes ok
7.  close the modal
8. reopen the edit settings modal
7. right click on the logo now and copy the image url, open it in a new tab , should be an s3 image url
8. change the logo again (steps 3-6)
9. reload the other tab, that image should be gone/deleted now
